externs/shaka/player.js

```diff
@@ -387,6 +389,9 @@ shaka.extern.TrackList;
  *   The minimum bandwidth of a variant track, in bit/sec.
  * @property {number} maxBandwidth
  *   The maximum bandwidth of a variant track, in bit/sec.
+ * @property {number} maxDisabledTime
+ *   The maximum time a variant can be disabled when NETWORK HTTP_ERROR
+ *   is reached, in seconds.
```

Comment:
You need to add any new config values to `demo/config.js`, so that they can be configured in the demo.
You'll also need new localized string entries for the demo, so that will be changes in `demo/common/message_ids.js`, `demo/locales/en.json`, and `demo/locales/source.json`.